inductiva/molecules/scenarios/protein_solvation/README.md

```diff
@@ -0,0 +1,44 @@
+# Protein Solvation Scenario
+
+This scenario models the dynamics of a protein solvated in water inside a cubic box. The system is modelled with the motion equations for each individual atoms according to the prescription of a molecular force field approximation.
+
+To initilaize the scenario, the user can define the following parameters:
+-- Protein .pdb file: PDB file containing the structure of the protein to simulate;
+-- Temperature of the system in K.
+
+Now, the user is prepared to simulate protein dynamics. Please note that the scenario currently supports only PDB files as input. We have conducted extensive testing with structures generated by the AlphaFold2 model and structures from the RCSB database. Before simulation, files undergo pre-processing to strip away all non-protein components of the system, such as ligands, non-standard residues, or water molecules, leaving only the protein.
```

Comment:
Can you add a link to the databases and the AlphaFold model. The idea is that people can test really quickly :)

Reply:
@david451lima https://alphafold.ebi.ac.uk/